agent/src/agent/cli/prompt/pipeline/schemaless.py

```diff
@@ -76,12 +76,13 @@ def set_values(self):
             raise click.UsageError('Set value properties or count records flag')
 
     @infinite_retry
-    def set_measurement_names(self):
-        prompt_text = 'Measurement names' if self.config.get('static_what', True) else 'Measurement properties names'
-        self.config['measurement_names'] = self.prompt_object(
-            prompt_text + '. Example -  property:measure property2:measure2',
-            self.get_default_object_value('measurement_names')
-        )
+    def prompt_measurement_names(self):
+        if self.config.get('static_what', True):
+            prompt_text = 'Metric names. Example -  original_name1:new_name1 original_name2:new_name2',
```

Comment:
I'd keep field1 and field2 from the values prompt because I think `original_name` is confusing. (Same for non-static what) 
```Metric names. Example - field1:new_name1 field2:new_name2```